cmd/troubleshoot/cli/run.go

```diff
@@ -182,6 +182,7 @@ func runTroubleshoot(v *viper.Viper, arg string) error {
 		ProgressChan:              progressChan,
 		SinceTime:                 sinceTime,
 		FromCLI:                   true,
+		Redact:                    true,
```

Comment:
Shouldn't this read value from the CLI flag?

Reply:
When we want to submit a pull request upstream, yes (flag or environment variable). The existing `--redact` flag has a different meaning and is not fully implemented.
For our current need this is sufficient, because we use it as a library and can initialize `SupportBundleCreateOpt` with redact set to false.
I wanted to wait on Replicated's feedback on the existing CLI flag before adding one that potentially conflicts with their plan. What do you think?